experiment-rollout.js

```diff
@@ -44,29 +44,25 @@ const data = {
         experimentType: 1,
         rolloutType: 1,
         priority: [
-            {
-                status: 1,
-                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
-            },
             {
                 status: 2,
                 name: 'Brands'
             },
             {
                 status: 2,
-                name: 'Discord staff'
+                name: 'Discord Staff'
             },
             {
-                status: 1,
-                name: 'Partnered and verified server owners (Includes old ones + some content creators)'
+                status: 2,
+                name: 'Partnered, subscription-enabled (100USD/mo+), and verified server owners (includes some old partners + some content creators)'
             },
             {
                 status: 1,
-                name: 'Moderator programs alumnis'
+                name: 'Verified bots (Server count wins name conflicts, other ones will get their discriminator appended)'
```

Comment:
Where did you get this info? (I mean the reorder)

Reply:
From the blog post, and messages from employees. Discord is giving access to Pomelo as a risk-mitigation not as a reward. And it's ordered by risk of impersonation, brands being more likely to be impersonated than Discord Staff, Discord Staff being more likely than Partners, etc..